listenbrainz/webserver/views/login.py

```diff
@@ -29,7 +30,7 @@ def musicbrainz_post():
     if provider.validate_post_login():
         user = provider.get_user()
         db_user.update_last_login(user.musicbrainz_id)
-        login_user(user)
+        login_user(user, remember=True, duration=datetime.timedelta(days=config.days))
```

Comment:
please use `current_app.config['days']` here instead of importing config explicitly.